src/constructs/core/parameters/sns.ts

```diff
@@ -0,0 +1,54 @@
+import { SSM_PARAMETER_PATHS } from "../../../constants/ssm-parameter-paths";
+import { isSingletonPresentInStack } from "../../../utils/singleton";
+import type { GuStack } from "../stack";
+import { GuStringParameter } from "./base";
+
+/**
+ * Creates a CloudFormation parameter which references an SNS topic to send
+ * alarms to. By default, the bucket name is stored in an SSM Parameter called
+ * `/account/services/alarm.topic.name`.
+ */
+export class GuAlarmTopicParameter extends GuStringParameter {
+  private static instance: GuAlarmTopicParameter | undefined;
+
+  private constructor(scope: GuStack) {
+    super(scope, "AlarmTopicName", {
+      description: SSM_PARAMETER_PATHS.AlarmTopic.description,
+      default: SSM_PARAMETER_PATHS.AlarmTopic.path,
+      fromSSM: true,
+    });
+  }
+
+  public static getInstance(stack: GuStack): GuAlarmTopicParameter {
+    if (!this.instance || !isSingletonPresentInStack(stack, this.instance)) {
+      this.instance = new GuAlarmTopicParameter(stack);
+    }
+
+    return this.instance;
+  }
+}
+
+/**
+ * Creates a CloudFormation parameter which references an SNS topic to send
+ * notifications to. By default, the bucket name is stored in an SSM Parameter
+ * called `/account/services/alarm.topic.name`.
+ */
```

Comment:
> By default, the bucket name is stored

I think this doc string is out of date? (same for `GuAlarmTopicParameter` above).

Should we want to make reference to [`GuAnghammaradTopicParameter`](https://github.com/guardian/cdk/blob/147ad77533ca384ef504cc0719aa87d8c84742be/src/constructs/core/parameters/anghammarad.ts) here?

Reply:
@akash1810 ah bucket is a copy+paste+edit error! Thanks. Re Anghammarad, yes good shout - wil mirror the original parameter description.